include/onnxruntime/core/framework/tensor.h

```diff
@@ -69,10 +69,29 @@
   /// <param name="strides"></param>
   static void InitOrtValue(MLDataType p_type, const TensorShape& shape,
                            void* p_data, const OrtMemoryInfo& location,
-                           OrtValue& ort_value, ptrdiff_t offset = 0, gsl::span<const int64_t> strides = {});
+                           OrtValue& ort_value, ptrdiff_t offset = 0,
+                           gsl::span<const int64_t> strides = {});
```

Comment:
>  gsl::span<const int64_t> strides = {}); [](http://example.com/codeflow?start=25&length=41)

Should strides be a part of this PR?

Reply:
We didn't add strides. i think this line is just an auto-format change.